hbase-client/src/main/java/org/apache/hadoop/hbase/client/Admin.java

```diff
@@ -2634,4 +2634,9 @@ List<LogEntry> getLogEntries(Set<ServerName> serverNames, String logType, Server
    * Get the list of cached files
    */
   List<String> getCachedFilesList(ServerName serverName) throws IOException;
+
+  /**
+   * Clean BucketCache
+   */
```

Comment:
Nit: Let's be more specific. What does it mean to clean BucketCache? We are evicting everything? And why this is only for BucketCache, not for all BlockCache implementations?